### DISCUSSION TITLE
Error while compiling the project

### DISCUSSION THREAD

**Discussion by Bryan988** [dexidp/dex#2079] [Q&A] [answered]
Hello, 

I am facing an issue while compiling the project after clone. 

There's the error : `build github.com/dexidp/dex/cmd/dex: cannot load embed: malformed module path "embed": missing dot in first path element
make: *** [Makefile:33: bin/dex] Error 1`

Is it a bug ? Or do I need to fix something in my configs ?

**Comment by nabokihms** [comment, marked as answer]
It looks like you are using go below 1.16 version.

**Reply by Bryan988** [reply]
Yes ! Thank you !
